src/plone.server/plone/server/__init__.py

```diff
@@ -1,4 +1,11 @@
 # -*- encoding: utf-8 -*-
+# create logging
+import logging
+logger = logging.getLogger('plone.server')
+
+from zope.i18nmessageid import MessageFactory  # noqa
+_ = MessageFactory('plone')
+
```

Comment:
Can we move the message factory instance and logging instance to their own modules? This would be so we could address potential circular imports caused by having app_settings in the plone.server module.